app/src/ingest_bem_pdfs.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import sys
+import uuid
+from dataclasses import dataclass
+from typing import BinaryIO
+
+from smart_open import open as smart_open
+
+from src.adapters import db
+from src.app_config import app_config
+from src.db.models.document import Chunk, Document
+from src.ingestion.pdf_elements import EnrichedText
+from src.ingestion.pdf_postprocess import group_texts
+from src.util import pdf_utils
+from src.util.file_util import get_files
+from src.util.ingest_utils import process_and_ingest_sys_args
+from src.util.pdf_utils import Heading
+
+logger = logging.getLogger(__name__)
+
+# Print INFO messages since this is often run from the terminal
+# during local development
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+
+HEADER_PATTERN = r"(BEM\s\d*\s+\d+\sof\s\d+\s+\w.*)"
+
+
+def _get_bem_title(file: BinaryIO, file_path: str) -> str:
+    """
+    Get the BEM number from the file path (e.g., 100.pdf) and the
+    document title from the PDF meta data, then put the document
+    title in title case (e.g., INTRODUCTION EXAMPLE -> Introduction Example)
+    and combine: "BEM 100: Introduction Example"
+    """
+    pdf_title = pdf_utils.get_pdf_info(file)["title"]
+    bem_num = file_path.split("/")[-1].rsplit(".", 1)[0]
+    return f"BEM {bem_num}: {pdf_title}"
+
+
+def _ingest_bem_pdfs(
+    db_session: db.Session,
+    pdf_file_dir: str,
+    doc_attribs: dict[str, str],
+) -> None:
+    file_list = get_files(pdf_file_dir)
+    embedding_model = app_config.sentence_transformer
+
+    logger.info(
+        "Processing PDFs in %s using %s with %s", pdf_file_dir, embedding_model, doc_attribs
+    )
+    for file_path in file_list:
+        if not file_path.endswith(".pdf"):
+            return
+
+        logger.info("Processing file: %s", file_path)
+        with smart_open(file_path, "rb") as file:
+            grouped_texts = _parse_pdf(file)
+
+            doc_attribs["name"] = _get_bem_title(file, file_path)
+            document = Document(content=str(grouped_texts), **doc_attribs)
```

Comment:
`content` doesn't seem correct to me, I think we want this to be something like:
```
content="\n".join(g.text for g in grouped_texts)
```